### ISSUE TITLE
Errors when trying to use this module. 

### ISSUE THREAD

**Issue by JeanHules** [namannehra/flipping-pages#3]
14:27  error    Unable to resolve path to module 'flipping-pages'  import/no-unresolved
14:27  error    Missing file extension for "flipping-pages"        import/extensions

I've read all of the suggested fixed to this error and when implemented, it breaks other modules. Any idea why this particular module breaks?

**Comment by namannehra** [comment]
I can confirm that it works with create-react-app. The problem might have something something to do with your webpack config. Can you post the steps to reproduce the problem along with you config files?

**Comment by JeanHules** [comment]
I simply did a yarn add and added the import link in the JS file.

Webpack Config:
```
const path = require('path');
const webpack = require('webpack');
const dotenv = require('dotenv');
const ExtractTextPlugin = require('extract-text-webpack-plugin');
const MergeIntoSingle = require('webpack-merge-and-include-globally');

module.exports = () => {
	const env = dotenv.config().parsed;
	const envKeys = Object.keys(env).reduce((prev, next) => {
		prev[`process.env.${next}`] = JSON.stringify(env[next]);
		return prev;
	}, {});

	return {
		context: __dirname,
		entry: [
			'babel-polyfill',
			'react-hot-loader/patch',
			'webpack-dev-server/client?http://localhost:8081',
			'webpack/hot/only-dev-server',
			'./js/ClientApp.jsx'
		],
		devtool: 'cheap-eval-source-map',
		output: {
			path: path.join(__dirname, 'public'),
			filename: 'bundle.js',
			publicPath: '/public/'
		},
		devServer: {
			hot: true,
			publicPath: '/public/',
			historyApiFallback: true,
			headers: {
				'Access-Control-Allow-Origin': '*',
				'Access-Control-Allow-Headers': 'Origin, X-Requested-With, Content-Type, Accept'
			}
		},
		resolve: {
			extensions: ['.js', '.jsx', '.json']
		},
		stats: {
			colors: true,
			reasons: true,
			chunks: true
		},
		plugins: [
			new webpack.HotModuleReplacementPlugin(),
			new webpack.NamedModulesPlugin(),
			new webpack.DefinePlugin(envKeys),
			new MergeIntoSingle({
				files: {
					'recruitme-vendor.js': ['./public/js/*']
				}
			})
		],
		module: {
			rules: [
				{
					enforce: 'pre',
					test: /\.jsx?$/,
					loader: 'eslint-loader',
					exclude: /node_modules/
				},
				{ test: /\.(js|jsx)$/, exclude: /node_modules/, loader: 'babel-loader' },
				{ test: /\.(png|woff|woff2|eot|ttf|svg)$/, loader: 'url-loader?limit=100000' },
				{
					test: /\.scss$/,
					use: [
						{
							loader: 'style-loader'
						},
						{
							loader: 'css-loader'
						},
						{
							loader: 'sass-loader'
						}
					]
				}
			]
		}
	};
};
```

**Comment by namannehra** [comment]
I think the errors are thrown by ESLint. Check if removing the `eslint-loader` resolves the issue.

**Comment by namannehra** [comment]
Also make sure 'flipping-pages' is in `dependencies` and not `devDependencies`

**Comment by JeanHules** [comment]
It seems to work with eslint-loader removed, but I would like to keep eslint. Can you think of a work around?

**Comment by namannehra** [comment]
`flipping-pages` is in `dependencies` and not `devDependencies`?

**Comment by JeanHules** [comment]
it is listed in dependencies.

**Comment by namannehra** [comment]
Can you post the file that imports `flipping-pages`?

**Comment by JeanHules** [comment]
Here is the import section:
```
// @flow

import React, { Component } from 'react';
import PropTypes from 'prop-types';
import axios from 'axios';
import * as Scroll from 'react-scroll';
import { connect } from 'react-redux';
import Select from 'react-select';
import { EditorState, convertToRaw, ContentState } from 'draft-js';
import { SortableContainer, SortableElement, arrayMove } from 'react-sortable-hoc';
import draftToHtml from 'draftjs-to-html';
import FlippingPages from 'flipping-pages';
import htmlToDraft from 'html-to-draftjs';
import SweetAlert from 'react-bootstrap-sweetalert';
import { Editor } from 'react-draft-wysiwyg';
import SideNav from './components/SideNav';
import ChapterSection from './components/ChapterSection';
import ChapterListing from './components/ChapterListing';
import { setCurrentModal, setQuiz } from './actionCreators';
```

Here is my ESLint config file:
```
{
	"extends": ["plugin:flowtype/recommended", "airbnb", "prettier", "prettier/react"],
	"plugins": ["flowtype", "prettier"],
	"parser": "babel-eslint",
	"parserOptions": {
		"ecmaVersions": 2016,
		"sourceType": "module",
		"ecmaFeatures": {
			"jsx": true
		}
	},
	"env": {
		"es6": true,
		"browser": true,
		"node": true,
		"jest": true
	}
}
```

**Comment by namannehra** [comment]
Try this:
Open `node_modules/flipping-pages/package.json`
Add this at the end `main: "FlippingPages.js"`

If this fixes the issue then I'll release an update with this.

**Comment by JeanHules** [comment]
That seemed to fix it. Thanks for the help!

**Comment by namannehra** [comment]
Fixed with 2.1.2
